core/lib/resources/presence/index.js

```diff
@@ -33,7 +33,7 @@ Presence.prototype.setup = function() {
   var self = this;
 
   this.manager = new PresenceManager(this.name, this.options.policy, Presence.sentry);
-  this.manager.on('user_online', function(userId, userType, userData) {
+  this.manager.on('user_online', function(userId, userType, userData, data) {
```

Comment:
We were talking about this yesterday with @bolddane Since he's already working on an abstraction for Message, we think that problem is probably going to go away soon.